src/components/DateRangePickerInput.jsx

```diff
@@ -45,6 +45,7 @@ const propTypes = forbidExtraProps({
   readOnly: PropTypes.bool,
   showCaret: PropTypes.bool,
   showDefaultInputIcon: PropTypes.bool,
+  inputIconPosition: PropTypes.oneOf('before', 'after'),
```

Comment:
`SimpleDatePickerShape`/`SimpleDatePicker` and `DateRangePickerShape`/`DateRangePicker` share same props..
But `DateRangePickerInputController` `SimpleDatePicker` `DateRangePickerInput`, they all have their own props, which are quite different from parents [`..Shape` and `..Picker`]..